tests/manifests/default/controller_def_1.3.3.yaml

```diff
@@ -1,325 +1,332 @@
 apiVersion: v1
```

Comment:
@juliev0 do we also want to autogenerate the `-copy1` file equivalent?

Reply:
no, that's okay, thank you!